src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,317 @@
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        },
+        {
+            "$ref": "mixins.json#/references"
+        }
+    ],
+    "dependentSchemas": {
+        "crispr_library_details": {
+            "not": {
+                "required": [
+                    "mpra_library_details"
+                ]
+            },
+            "comment": "Specification of crispr_library_details is mutually exclusive with specification of mpra_library_details."
+        },
+        "mpra_library_details": {
+            "not": {
+                "required": [
+                    "crispr_library_details"
+                ]
+            },
+            "comment": "Specification of mpra_library_details is mutually exclusive with specification of crispr_library_details."
+        },
+        "scope": {
+            "oneOf": [
+                {
+                    "required": [
+                        "targeted_genes"
+                    ],
+                    "properties": {
+                        "scope": {
+                            "enum": [
+                                "genes"
+                            ]
+                        }
+                    }
+                },
+                {
+                    "required": [
+                        "targeted_regions"
+                    ],
+                    "properties": {
+                        "scope": {
+                            "enum": [
+                                "chromosomal regions"
+                            ]
+                        }
+                    }
+                },
+                {
+                    "not": {
+                        "properties": {
+                            "scope": {
+                                "enum": [
+                                    "genes",
+                                    "chromosomal regions"
+                                ]
+                            }
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "accession": {
+            "accessionType": "FS"
+        },
+        "scope": {
+            "title": "Library Construct Scope",
+            "description": "The scope or scale that this construct library is designed to target. If the scope is across gene(s) or chromosomal region(s), these will need to be specified in the targeted_genes or targeted_regions property.",
+            "type": "string",
+            "enum": [
+                "genes",
+                "chromosomal regions",
+                "genome-wide",
+                "population"
```

Comment:
not sure about population? if we add it, then it should be explicitly explained in comment

Reply:
Removed for now, we can add back if given a definition by an IGVF lab